sample-config/.travis.yml

```diff
@@ -8,8 +8,9 @@ notifications:
 
 cache:
   directories:
-    - node_modules
-    - vendor
+    - $HOME/.composer/cache
```

Comment:
Not sure if YAML would prefer quotes around a string that could have spaces in it.

Reply:
I just copied it from Gutenberg 🤷‍♂️